plugins/jobs/server/models/job.py

```diff
@@ -52,19 +56,33 @@ def _validateStatus(self, status):
             raise ValidationException(
                 'Invalid job status %s.' % status, field='status')
 
-    def list(self, user=None, limit=0, offset=0, sort=None, currentUser=None):
+    def list(self, user, types=None, statuses=None,
```

Comment:
This could be purely my personal way of thinking. I see the `user` parameter as a filter, so `None` could mean filter value is `None` or no user filter (don't filter by user) because None has two level of meaning, in programming language it means something does exist, in our business logic, it means anonymous user.